inc/remove-admin-menu-items.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Remove admin menu pages.
+ *
+ * @package MEOM Dodo
+ */
+
+namespace MEOM\dodo;
+
+/**
+ * Remove admin menu items.
+ */
+function remove_admin_menu_items() {
+    /**
+     * Filters the hidden admin menu items.
+     *
+     * @since 1.0.3
```

Comment:
Is this version number related to the plugin version or is it something else?

Reply:
It's related in which version the filter is added. It's not super important in my opinion in these kind of small plugins.

Should I take it off?

Reply:
^ Ping @giraffewoods

Reply:
Was just thinking should it equal the bumped version number in composer.json / package.json if kept here

Reply:
It should, `1.0.2` as seen in this PR is my mistake coming from previous release. Once we release next one, I'll update the version numbers to `1.0.3`.

`package.json` version number doesn't matter though since this is not NPM package.

Reply:
If we keep the changelog updated, I don't think that `@since` is necessary. You can easily make errors while writing those :)

Reply:
Let's remove `@since`. It would be helpful for automating document pages but I don't think we are aiming for that.

Reply:
Done in db1929a.